WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/Cells/RefundItemTableViewCell.swift

```diff
@@ -0,0 +1,191 @@
+    ///
+    @IBOutlet private var itemCaption: UILabel!
+
+    /// Quantity button: Quantity to be refunded
+    ///
+    @IBOutlet private var itemQuantityButton: UIButton!
+
+    /// Needed to change it's axis with larger accessibility traits
+    ///
+    @IBOutlet private var itemsStackView: UIStackView!
+
+    /// Needed to make sure the `itemImageView` grows at the same ratio as the dynamic fonts
+    ///
+    @IBOutlet private var itemImageViewHeightConstraint: NSLayoutConstraint!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyCellStyles()
+        applyAccessibilityChanges()
+    }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        applyAccessibilityChanges()
+    }
+}
+
+// MARK: View Styles Configuration
+private extension RefundItemTableViewCell {
+    func applyCellStyles() {
+        applyCellBackgroundStyle()
+        applyItemImageStyles()
+        applyLabelsStyles()
+        applyRefundQuantityButtonStyle()
+    }
+
+    func applyItemImageStyles() {
+        itemImageView.layer.borderWidth = Constants.itemImageViewBorderWidth
+        itemImageView.layer.borderColor = UIColor.border.cgColor
+    }
+
+    func applyCellBackgroundStyle() {
+        applyDefaultBackgroundStyle()
+    }
+
+    func applyLabelsStyles() {
+        itemTitle.applyBodyStyle()
+        itemCaption.applyFootnoteStyle()
+    }
+
+    func applyRefundQuantityButtonStyle() {
+        itemQuantityButton.applySecondaryButtonStyle()
+        itemQuantityButton.titleLabel?.applyBodyStyle()
+        itemQuantityButton.contentEdgeInsets = Constants.quantityButtonInsets
+    }
+}
+
+// MARK: Accessibility
+private extension RefundItemTableViewCell {
+    func applyAccessibilityChanges() {
+        adjustItemsStackViewAxis()
+        adjustItemImageViewHeight()
+    }
+
+    /// Changes the items stack view axis depending on the view `preferredContentSizeCategory`.
+    ///
+    func adjustItemsStackViewAxis() {
+        itemsStackView.axis = traitCollection.preferredContentSizeCategory > .accessibilityMedium ? .vertical : .horizontal
+    }
+
+    /// Changes the items image view height acording to the current trait collection
+    ///
+    func adjustItemImageViewHeight() {
+        itemImageViewHeightConstraint.constant = UIFontMetrics.default.scaledValue(for: Constants.itemImageViewHeight, compatibleWith: traitCollection)
+    }
+}
+
+// MARK: ViewModel Rendering
+extension RefundItemTableViewCell {
+
+    /// Configure cell with the provided view model
+    ///
+    func configure(with viewModel: RefundItemViewModel) {
+        itemTitle.text = viewModel.productTitle
+        itemCaption.text = viewModel.productQuantityAndPrice
+        itemQuantityButton.setTitle(viewModel.quantityToRefund, for: .normal)
+
+        if let _ = viewModel.productImage {
+            // TODO: fill product image
```

Comment:
is the image set up in a future PR?

Reply:
Yes, this is meant to go in a future PR! More specifically here https://github.com/woocommerce/woocommerce-ios/issues/2842